src/server.py

```diff
@@ -91,29 +93,35 @@ def ref(self, sound):
                           sound_event_id=sound.sound_event_id,
                           uuid=sound.uuid)
 
-    def unref(self, sound, count=1):
-        if sound.uuid not in self.registry.refcount:
-            self.logger.warning("This uuid is not registered in the refcount "
-                                "registry.", uuid=sound.uuid)
+    def unref(self, sound, clear_all=False):
+        try:
```

Comment:
I think it's fine to remove the `try` block. Assertion errors should just propagate up to the top and not be caught.